platform/security/sts/security-sts-ldapclaimshandler/src/main/java/ddf/security/sts/claimsHandler/RoleClaimsHandler.java

```diff
@@ -87,18 +84,6 @@ public RoleClaimsHandler(AttributeMapLoader attributeMapLoader) {
     this.attributeMapLoader = attributeMapLoader;
   }
 
-  public URI getRoleURI() {
-    URI uri = null;
-    try {
-      uri = new URI(roleClaimType);
-    } catch (URISyntaxException e) {
-      LOGGER.info(
-          "Unable to add role claim type. Set log level for \"ddf.security.sts.claimsHandler\" to DEBUG for more information.");
-      LOGGER.debug("Unable to add role claim type.", e);
-    }
-    return uri;
-  }
-
   public String getPropertyFileLocation() {
     return propertyFileLocation;
   }
```

Comment:
❓ Why does the `RoleClaimsHandler` care about the property file? Isn't the `PropertyFileClaimsHandler` responsible for this?

Reply:
This is a different properties file.